src/dmtcpplugin.cpp

```diff
@@ -216,81 +195,13 @@ dmtcp_set_ckpt_file(const char *filename)
 EXTERNC const char *
 dmtcp_get_ckpt_filename(void)
 {
-  static string *filename =
-    new string(ProcessInfo::instance().getCkptFilename());
-  return filename->c_str();
+  return ProcessInfo::instance().getCkptFilename().c_str();
 }
 
 EXTERNC const char *
 dmtcp_get_ckpt_files_subdir(void)
 {
-  static string *tmpdir =
-    new string(ProcessInfo::instance().getCkptFilesSubDir());
-  return tmpdir->c_str();
-}
-
-EXTERNC void
-dmtcp_get_bin_dir(char dmtcp_bin_dir[]) {
```

Comment:
@gc00 was the one who put all the functions related to `dmtcp_get_libc_addr` in #837. I would like to get his opinion on removing these functions. We can move them or comment out temporarily.

Reply:
See my earlier comment on the reasons for keeping `dmtcp_get_libc_addr`.  Sometimes, we really want the low-level glibc fnc, with no interposition.  We either need this or an equivalent mechanmism.